base/CMakeLists.txt

```diff
@@ -110,6 +110,8 @@ target_include_directories(${target} PUBLIC
 
   # DELETE ME ONCE USING root targets
   ${ROOT_INCLUDE_DIR}
+
+  ${VMC_INCLUDE_DIRS}
```

Comment:
Are you sure this is needed?

If the imported `VMCLibrary` target does not define its `INTERFACE_INCLUDE_DIRECTORIES` property, we should define it on this level, see e.g. https://github.com/FairRootGroup/FairRoot/blob/4ab2d2b1f0392ad93b92026d70fe049c38665f7d/CMakeLists.txt#L271-L273 on how to do this right after the `find_package2` call in the top-level `CMakeLists.txt`.

Adding these kind of variables in each an every `CMakeLists.txt` is not good because it duplicates code and encourages the copy&paste mess we had in the past - ideally, a dependency of a library or executable should only be declared once per `CMakeLists.txt` in the `target_link_libraries` call.

Reply:
removed VMC_INCLUDE_DIRS from sub-folders CMakeLists